README.md

```diff
@@ -95,6 +95,33 @@ because they need additional setup on your system:
 
       pip install slycot
 
+  Note that building Slycot might fail for the following reasons:
+  * The Slycot package contains a cmake check which fails when it
+    detects multiply NumPy include directories. This will cause the
+    build to fail in venvs with any Python interpreter that has NumPy
+    globally installed.
+    To circumvent this problem, use another Python interpreter. If
+    you do not want to build CPython yourself, you can use
+    [pyenv](https://github.com/pyenv/pyenv) or
+    [uv](https://github.com/astral-sh/uv) to easily install another
+    interpreter.
```

Comment:
[mise](https://mise.jdx.dev/) could also be suggested.

Reply:
Done.